workers/miscale.py

```diff
@@ -1,3 +1,6 @@
+from math import floor
+import json
```

Comment:
I think this can be removed, mqtt lib should handle serialization on its own.

Reply:
You mean json.dump()?

Reply:
If it, MQTT serialize payload only when using MqttMessageConfig, in MqttMessage there is no serialization and it send error that payload should be string, not object!

Reply:
Screen of MqttMessageConfig:
<img width="597" alt="CleanShot 2020-01-04 at 16 56 21@2x" src="https://user-images.githubusercontent.com/7906418/71765175-259aaf80-2f13-11ea-8750-422fdccd88fb.png">

Screen of MqttMessage:
<img width="489" alt="CleanShot 2020-01-04 at 16 56 43@2x" src="https://user-images.githubusercontent.com/7906418/71765180-32b79e80-2f13-11ea-85e6-2ffcf22da668.png">

Reply:
Oh you are right. I need to write this better, probably MqttMessage should check message type and do json dump on his own, instead of need to do it in each place. I can clear that later.